<launch>        
  <!-- 
       Start an apriltag node using the default settings
       to connect to an OpenNI device (e.g. Kinect). 
  -->
  <node pkg="apriltags" type="apriltags" name="apriltags">
    <param name="tag_size" value="0.037" />
    <param name="tf_frame" value="/openni_rgb_optical_frame"/>

    <remap from="image" to="/camera/rgb/image_rect"/>
    <remap from="camera_info" to="/camera/rgb/camera_info"/>
    <remap from="marker_array" to="/apriltags/marker_array"/>
  </node>
</launch>